# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lilo/device-mapper-cross.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

During cross builds the checks need to be run against the new system root.

  - Rene Rebe <rene@exactcode.de>

--- lilo-22.7.3/Makefile.vanilla	2006-10-21 19:29:58.000000000 +0000
+++ lilo-22.7.3/Makefile	2006-10-21 19:30:40.000000000 +0000
@@ -98,9 +98,9 @@
   cat $$ROOT/etc/lilo.defines; else echo $(CONFIG); fi ) | \
   sed 's/-D/-DLCF_/g'`
 LILO=-DLILO=0x73141bae
-KVER=`if [ -f /usr/include/linux/version.h ]; then echo -DHAS_VERSION_H; fi`
-HDM=`if [ -f /usr/include/libdevmapper.h ]; then echo -DHAS_LIBDEVMAPPER_H; fi`
-DEVMAPPER=`if [ -f /usr/include/libdevmapper.h ]; then echo "-ldevmapper"; fi`
+KVER=`if [ -f $(ROOT)/usr/include/linux/version.h ]; then echo -DHAS_VERSION_H; fi`
+HDM=`if [ -f $(ROOT)/usr/include/libdevmapper.h ]; then echo -DHAS_LIBDEVMAPPER_H; fi`
+DEVMAPPER=`if [ -f $(ROOT)/usr/include/libdevmapper.h ]; then echo "-ldevmapper"; fi`
 
 TMP=_TmP_
 
